### PR TITLE
[7.3.x] JBPM-6047: Uploaded documents not visible on Documents tab on process Instance details

### PR DESCRIPTION
**Pull request by pefernan**
Copy of https://github.com/kiegroup/jbpm-form-modeler/pull/142

@hasys can you take a look?

This is part of an ensemble, merge with:
https://github.com/kiegroup/jbpm/pull/982
https://github.com/kiegroup/kie-wb-common/pull/1163
https://github.com/kiegroup/jbpm-form-modeler/pull/143
https://github.com/kiegroup/jbpm-wb/pull/874

### CODE REVIEW

File: jbpm-form-modeler-core/jbpm-form-modeler-service/jbpm-form-modeler-ui/src/main/java/org/jbpm/formModeler/components/renderer/FormRenderContextManagerImpl.java [hunk 1]
```diff
@@ -109,7 +109,7 @@ public FormRenderContext newContext(Form form, String deploymentId, Map<String,
 
     private FormRenderContext buildContext(String uid, Form form, String deploymentId, Map<String,Object> inputData, Map<String,Object> outputData, Map<String,Object> forms) {
         FormRenderContext ctx = new FormRenderContext(uid, form, inputData, outputData);
-        ctx.setDeploymentId( deploymentId );
+        ctx.setServerTemplateId(deploymentId );
```
Comment (mswiderski) [inline jbpm-form-modeler-core/jbpm-form-modeler-service/jbpm-form-modeler-ui/src/main/java/org/jbpm/formModeler/components/renderer/FormRenderContextManagerImpl.java]: This is bit confusing - setting deploymentId as server template...

Comment (pefernan) [reply jbpm-form-modeler-core/jbpm-form-modeler-service/jbpm-form-modeler-ui/src/main/java/org/jbpm/formModeler/components/renderer/FormRenderContextManagerImpl.java]: @mswidersky well seen... Forgot to rename the parameter. DeploymentId is no longer used, so I changed it.

**Comment by pefernan** [comment]
@mswiderski fixed ;)

**Comment by jsoltes** [comment]
jenkins execute full downstream build

**Comment by pefernan** [comment]
jenkins execute full downstream build

**Pull request merged**